src/odemis/driver/andorcam2.py

```diff
@@ -2954,6 +2954,8 @@ def GetMostRecentImage16(self, cbuffer, size):
         ndbuffer[...] = self._data[self.roi[2] - 1:self.roi[3]:self.binning[1],
                                    self.roi[0] - 1:self.roi[1]:self.binning[0]]
 
+        ndbuffer += (numpy.random.rand(res[1], res[0]) * 1000 - 500).astype(numpy.uint16)
```

Comment:
Please check that all the test cases still pass (andorcam2_test). I don't think any test case expects that images are always the same, but you need to double check.

Also, I'd suggest to tune down a little bit the values to around 200, which is the typical noise level on these cameras. Also make them only positive, to simplify a little bit. Moreover `randint()` seems to do just what you need. So you could write:
`ndbuffer += numpy.random.randint(0, 200, ndbuffer.shape, dtype=ndbuffer.dtype)`

Reply:
The Andorcam tests run fine.

Reply:
I force-pushed it now.